recipes/libproperties/all/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import ConanFile, CMake, tools
+
+
+class LibpropertiesConan(ConanFile):
+    name = "libproperties"
+    version = "1.1.2"
+    license = "Apache-2.0 License"
+    author = "libbylg@126.com"
+    url = "https://github.com/tinyhubs/libproperties"
+    description = "libproperties is a library to parse the Java .properties files. It was writen in pure C. And fullly compatible with the Java .properties file format."
+    topics = (".properties", "Java", "pure C")
```

Comment:
```suggestion
    topics = ("properties", "java", "pure-c")
```

Make it easier to be find by search engine